### PR TITLE
Feat/gpt page

### PR DESCRIPTION
**Pull request by sai-tanush**

### CODE REVIEW

File: src/components/GptMovieSuggesstions.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { RootState } from "../utils/appStore";
+import MovieList from "./MovieList";
+
+const GptMovieSuggesstions: React.FC = () => {
+  const { movies, gptMovies } = useSelector((store: RootState) => store.gpt);
+  if (!gptMovies) return null;
+
+  return (
+    <div className="p-4 mt-2 mx-3 bg-black text-white bg-opacity-70">
+      <div>
```
Comment (blazing-Onion) [inline src/components/GptMovieSuggesstions.tsx]: why use an extra div? There are a lot of unnecessary nested divs

File: src/components/GptSearchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        movie +
+        "&include_adult=false&language=en-US&page=1",
+      API_OPTIONS
+    );
+    const jsonData = await data.json();
+
+    return jsonData.results;
+  };
+
+  const handleGotSearchClick = async () => {
```
Comment (blazing-Onion) [inline src/components/GptSearchBar.tsx]: It's an event click, pretty sure adding an async here wouldn't make much difference

File: src/utils/geminiai.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { GoogleGenerativeAI } from "@google/generative-ai";
```
Comment (blazing-Onion) [inline src/utils/geminiai.ts:1]: these act more like services, Create a folder for services which will have firebase, gemini and other configurations setup

File: src/components/GptMovieSuggesstions.tsx (hunk 1 above)
Comment (sai-tanush) [reply src/components/GptMovieSuggesstions.tsx]: resolved in this component

File: src/utils/geminiai.ts (hunk 3 above)
Comment (sai-tanush) [reply src/utils/geminiai.ts:1]: i create another branch for folder structure

File: src/components/GptSearchBar.tsx (hunk 2 above)
Comment (sai-tanush) [reply src/components/GptSearchBar.tsx]: I am using await keyword inside

**Review by blazing-Onion — APPROVED** [review]

**Pull request closed without merging**